src/main/resources/com/google/api/codegen/nodejs/package.json.snip

```diff
@@ -40,7 +40,8 @@
     },
     "scripts": {
       "publish-module": "node ../../scripts/publish.js {@metadata.shortName}",
-      "test": "mocha test/*.js"
+      "test": "mocha test/*.js",
+      "smoke-test": "mocha smoke-test/*.js --timeout 5000"
```

Comment:
Nit. move 'smoke-test' to be before 'test' in order to follow alphabetical order.